assets/js/atomic/blocks/product-elements/button/style.scss

```diff
@@ -50,6 +61,17 @@
 		margin-left: auto;
 		white-space: normal;
 		word-break: break-word;
+		width: 150px;
```

Comment:
Nit: we could add `overflow: hidden` here. Otherwise it felt to me like the text was coming from "outside" the button.